.env

```diff
@@ -74,5 +74,5 @@ REACT_APP_MOCK=true
 REACT_APP_LOCALES="locales"
 
 # IPFS uploading
-REACT_APP_PINATA_API_KEY=
-REACT_APP_PINATA_SECRET_API_KEY=
+REACT_APP_PINATA_API_KEY=foo
+REACT_APP_PINATA_SECRET_API_KEY=bar
```

Comment:
Nice try, but nope :)

I rather untrack this file than putting trash defaults.
If we do that it'll defeat the purpose of the warning

Reply:
Good catch, I wasn't planing on committing this, I thought this file is ignored :smile: